openedx/adg/lms/conftest.py

```diff
@@ -21,3 +28,54 @@ def user_with_user_profile(request):
     user = UserFactory()
     UserProfileFactory(user=user)
     return user
+
+
+@pytest.fixture(name='user_client')
+def user_client_login(request, client):
+    """
+    User and client login fixture. User will be authenticated for all tests where we pass this fixture.
+    """
+    user = UserFactory()
+    client.login(username=user.username, password='test')
+    return user, client
+
+
+@pytest.fixture(name='courses')
+def course_overviews(current_time):
+    """
+    Fixture which returns multiple courses
+    """
+    course_start_end_date = {
+        'start_date': current_time - timedelta(days=1),
+        'end_date': current_time + timedelta(days=1),
+    }
+    return {
+        'test_course1': CourseOverviewFactory(language='en', **course_start_end_date),
+        'test_course2': CourseOverviewFactory(language='ar', **course_start_end_date),
+    }
+
+
+@pytest.fixture(name='prereq_course_groups')
+def create_prerequisite_course_group(request):
+    """
+    This fixture creates courses equal to the size of the boolean list and makes them prerequisite depending on the
+    boolean value at each index
+    """
```

Comment:
Can we please update this docstring as I think we are not passing a boolean list rather an int and str, plus lets also mention that incase 'business_line_pre_req' is passed, the courses in the course groups are made common business line prerequisite otherwise program prerequisite